addons/digest/__init__.py

```diff
@@ -3,3 +3,13 @@
 
 from . import controllers
 from . import models
+
+from odoo.api import Environment, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    env = Environment(cr, SUPERUSER_ID, {})
```

Comment:
Don't you put a TODO or something for saying this shouldn't be backported and put the dependency instead?

Reply:
@pedrobaeza  PR for master: https://github.com/odoo/odoo/pull/28239

Reply:
Yeah, but if not specified,  forward-port team will keep this as well in master

Reply:
I could but chances are it will get forward-ported anyway and even if it is, it's just dead code

Reply:
Well, if you put a `TODO: To be removed in >=v13`, at least someone reviewing later can remove the dead code.